### DISCUSSION TITLE
This is a really perfect feature in GitHub!

### DISCUSSION THREAD

**Discussion by skyleaworlder** [community/community#2874] [Discussions] [answered]
I like it!

**Comment by evi-liu** [comment, marked as answer]
Glad you're loving it! What's your favorite part so far and why do you think it's perfect for GitHub?

**Reply by skyleaworlder** [reply]
Thanks for your kindly reply.
Discussions make issues be the ISSUES. 

There would be lots of discussions about environment, runtime errors and so on in issues part before. Discussions provide a perfect platform to share and solove problems for maintainers, developers, contributors and cloners.

I believe that issues part will be purified, to give more feature drafts, TODO List / RoadMap advice.
